### PR TITLE
Releases explanation

### PR DESCRIPTION
**Pull request by tomvanmele**
### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [ ] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [x] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [ ] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas.datastructures.Mesh`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: docs/releases.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+.. rst-class:: lead
+
+COMPAS releases are available on the Python Package Index (PyPI), on ``conda-forge``
+or directly from the Github repo.
+
+Versions are tagged following the Semantic Versioning (semver) convention of ``MAJOR.MINOR.PATCH``.
+For more information see `Semantic Versioning 2.0.0 <https://semver.org/>`_.
+
+The current version of COMPAS is ``0.19.1``, which means we have not yet established a compleltely stable API.
```
Comment (tetov) [inline docs/releases.rst]: compleltely -> completely

File: docs/releases.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+However, the ``0.19`` version series is the last before the release of ``COMPAS 1.0.0``!
+
+In the future, in addition to a version number, major releases will also have a name following the pattern
+``<geometric adjective> <nautical term>``, with the words forming an alliteration and the alliteration letter correspoding to the number of the release.
```
Comment (beverlylytle) [inline docs/releases.rst]:
```suggestion
``<geometric adjective> <nautical term>``, with the words forming an alliteration and the alliteration letter corresponding to the number of the release.
```
Also, I just learned this, but technically (pedantically) we are asking for some thing that is not only an alliteration but also a tautogram. https://en.wikipedia.org/wiki/Tautogram

**Comment by gonzalocasas** [comment]
I suggest we change the tenses so that this doesn't need to be changed again in a few weeks, eg. 

```
The API of ``COMPAS`` was under heavy development up until ``COMPAS 0.19``,
the last version series before ``1.0``.

Starting with ``1.0`` the API has been stabilized and the projects adheres strictly to
semver. In addition to the version number, major releases also have a name
following the pattern ``<geometric adjective> <nautical term>``, with the words
forming an tautogram in which the first letter increases alphabetically on every major
release and rolls over back to A after reaching Z.
```

(also included the tautogram term and the rollover to A of the Slack conversation)

File: docs/releases.rst [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+For example, the first release could be ``COMPAS 1.0.0: the Affine Admiral``.
+The second could be ``COMPAS 2.0.0: the Barycentric Bulkhead``, and so on.
```
Comment (gonzalocasas) [inline docs/releases.rst]: I'd drop the definite article and also the patch number:
```suggestion
For example, the first release could be ``COMPAS 1.0: Affine Admiral``.
The second could be ``COMPAS 2.0: Barycentric Bulkhead``, and so on.
```

**Review by gonzalocasas — APPROVED** [review]
LGTM!

**Pull request merged**